code/weapon/weapons.cpp

```diff
@@ -3078,6 +3078,14 @@ int parse_weapon(int subtype, bool replace, const char *filename)
 				stuff_vec3d(&t5info->end_pos_rand);
 			}
 
+			if (optional_string("+Slash completion over beam lifetime curve:")) {
```

Comment:
It's not a huge deal, but this name is a little odd to me, and I'm wondering if there's a better one. How do you feel about 'Slash interpolation..."? It's more 'jargony' but feels more specific, slash "completion" feels like it isn't clear what precisely it refers to.

Reply:
How about "position"? I think "interpolation" relies on a certain way of thinking about the beam's behavior and the underlying code, and I worry that a lot of people just won't have any idea what it means in this situation.

Reply:
I'm ok with "position"